nomos-blend/network/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 cached = "0.53.1"
 futures = "0.3.30"
 futures-timer = "3.0.3"
-libp2p = "0.53"
+libp2p = "0.55"
```

Comment:
To use all functions of [`libp2p-allow-block-list`](https://docs.rs/libp2p-allow-block-list/latest/libp2p_allow_block_list/), we need to upgrade libp2p to 0.55: https://github.com/libp2p/rust-libp2p/commit/e63975d7742710d4498b941e151c5177e06392ce.